packages/environment-check/src/utils.ts

```diff
@@ -0,0 +1,159 @@
+import type { WorkspaceFolder } from 'vscode';
```

Comment:
the name `utils.ts` is not really specific on the purpose. Can we split this code into functions that belong together and name the file appropriately?